zadaca5/src/zadatak1/Performance.java

```diff
@@ -0,0 +1,71 @@
+package zadatak1;
+
+import java.util.Scanner;
+
+public class Performance {
+    private int KPIScore;
+    private double coefficient;
+    private double brutoSalary;
+
+    public Performance () {
+        this.enterValues();
+    }
+    public Performance (int KPIScore, double brutoSalary) {
+        this(KPIScore,brutoSalary,0.5);
+    }
+    public Performance (int KPIScore, double brutoSalary, double coefficient) {
+        this.KPIScore = KPIScore;
+        this.brutoSalary =brutoSalary;
+        this.coefficient = coefficient;
+    }
+
+    public double getBrutoSalary() {
+        return brutoSalary;
+    }
+
+    public double getCoefficient() {
+        return coefficient;
+    }
+
+    public int getKPIScore() {
+        return KPIScore;
+    }
+
+    public void setBrutoSalary(double brutoSalary) {
+        this.brutoSalary = brutoSalary;
+    }
+
+    public void setCoefficient(double coefficient) {
+        this.coefficient = coefficient;
+    }
+
+    public void setKPIScore(int KPIScore) {
+        this.KPIScore = KPIScore;
+    }
+
+    public double bonusAmount(){
+        if (this.KPIScore >= 50){
+        return this.coefficient*this.KPIScore*this.brutoSalary/100;}
+        else return 0;
+    }
+
+    public void meetExpectations(){
+        if (this.KPIScore==100){
+            System.out.println("Congratulations, you have met the expectations, you will be rewarded with bounus amount of: " + this.bonusAmount());
+        }else if (this.KPIScore>100) {
+            System.out.println("Congratulations, you have exceeded the expectations, you will be rewarded with bounus amount of: " + this.bonusAmount());
+        } else if (this.KPIScore<100 && this.KPIScore>50) {
+            System.out.println("Unfortunately, you didn't met the expectations, but you haven't either underperformed. You will receive a bonus in amount of: " + this.bonusAmount()/2);
+        }else if (this.KPIScore<=50) {
+            System.out.println("Unfortunately, you have underperformed.");
+        }
+    }
+
+    public void enterValues(){
```

Comment:
This is cute, I like that you call this method from the constructor, but this is a bit of a double edged sword.

It is not obvious that calling a constructor without parameters, will prompt me to type in stuff. Also since this method is public I can, at any point, call it to change the whole state of a Performance object.

`enterValues()` doesn't really say much what those entered values do.

Think of it like this. When writing a class assume that the user will not be able to change your class at all, and ideally is able to use it without even looking at the source code.